Jenkinsfile

```diff
@@ -3150,51 +3160,51 @@ assert_frame_equal(training_curve, gt_curve, rtol=1e-3, atol=1e-3)"'''
         sh "rm -rf examples/nlp/language_modeling/gpt_index_mappings"
       }
     }
-    stage('L2: Megatron GPT with Rope Pretraining and Resume Training TP=2') {
```

Comment:
Do we need to remove this test?

Reply:
@ericharper this one uses `position_embedding_type=rope` which causes and error (TE bug).

Reply:
This test is not even using mcore_gpt or transformer_engine flags though....